common/spec/dependabot/pull_request_creator/message_builder/link_and_mention_sanitizer_spec.rb

```diff
@@ -200,6 +195,26 @@
           end
         end
       end
+
+      context "team mentions" do
+        let(:text) { "Thanks @dependabot/reviewers" }
+
+        it "sanitizes the team mention" do
+          expect(sanitize_links_and_mentions).to eq(
+            "<p>Thanks <code>@\u200Bdependabot/reviewers</code></p>\n"
+          )
+        end
+      end
+
+      context "multiple team mentions" do
```

Comment:
Worth still adding another test case where there's regular text between the team mention or modifying this test case?